### ISSUE TITLE
Touchpad scroll?

### ISSUE THREAD

**Issue by tv42** [graememcc/micropolisJS#33]
I have no idea how to make the map scroll(/pan) by just using a touchpad. Arrow keys seem to work.

**Comment by graememcc** [comment]
Yeah, I expect it's a pretty crummy experience on touch devices at the moment, I'm afraid - didn't have access to any back when I initially developed this.

Will try to address at some point, but unsure when that'll be.
